src/constants/env.js

```diff
@@ -1,3 +1,5 @@
 export const API_HOST = process.env.SAPPER_APP_API_HOST;
 export const AUTH_HOST = process.env.SAPPER_APP_AUTH_HOST;
 export const FRONTEND_BASE = process.env.SAPPER_APP_FRONTEND_BASE;
+export const ENV = process.env.SAPPER_APP_ENVIRONMENT;
```

Comment:
There is a `process.env.NODE_ENV` variable that is used in the *rollup.config.js*. Is it possible not to create more environment variables?

Reply:
Idk. I don't trust that variable, so I've created my own, the one that I can easily manipulate

Reply:
what exactly do you not trust about it? it is 'development' if running through `npm run dev`, and 'production' if running `npm run build`

Reply:
Well, I didn't know that and didn't bother finding out. Besides, we may set it to a heroku-specific value. I like control